docs/explanations/how_to_interpret_the_security_status_command.md

```diff
@@ -0,0 +1,100 @@
+# How to interpret the security-status command output
+
+The `security-status` command is used to get an overview
+of the packages installed in your machine. Currently,
+the command only support machine readable output: `json` or `yaml`.
+
+If you run the `ua security-status --format yaml` command on your
+machine, you are expected to see the output following this structure:
+
+```
+_schema_version: '0.1'
+summary:
+  num_esm_apps_packages: 0
+  num_esm_apps_updates: 0
+  num_esm_infra_packages: 1
+  num_esm_infra_updates: 1
+  num_main_packages: 70
+  num_multiverse_packages: 10
+  num_restricted_packages: 10
+  num_third_party_packages: 0
+  num_universe_packages: 9
+  num_installed_packages: 100
+  num_standard_security_updates: 0
+  ua:
+    attached: true
+    enabled_services:
+    - esm-apps
+    - esm-infra
+    entitled_services:
+    - esm-apps
+    - esm-infra
+packages:
+- origin: esm.ubuntu.com
+  package: zlib1g
+  service_name: esm-infra
+  status: upgrade_available
+  version: 1:1.2.8.dfsg-2ubuntu4.3+esm1
+```
+
+Let's understand what each key mean on the output of the `ua security-status` command:
+
+* **summary**: The summary of the system related to Ubuntu Advantage (UA) and
```

Comment:
A suggestion to consider, not sure if it would be significantly better

We could make field names inline code + bold like

```
**`summary`**
```
rendered:
**`summary`**